[2025-01-06T13:45:48.805+0100] {processor.py:186} INFO - Started process (PID=5142) to work on /Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-06T13:45:48.811+0100] {processor.py:914} INFO - Processing file /Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-06T13:45:48.812+0100] {logging_mixin.py:190} INFO - [2025-01-06T13:45:48.812+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-06T13:45:48.869+0100] {logging_mixin.py:190} INFO - [2025-01-06T13:45:48.863+0100] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-06T13:45:48.869+0100] {logging_mixin.py:190} INFO - [2025-01-06T13:45:48.869+0100] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-06T13:45:48.869+0100] {processor.py:927} WARNING - No viable dags retrieved from /Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-06T13:45:48.997+0100] {processor.py:208} INFO - Processing /Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.200 seconds
[2025-01-06T16:23:16.558+0100] {processor.py:186} INFO - Started process (PID=19809) to work on /Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-06T16:23:16.565+0100] {processor.py:914} INFO - Processing file /Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-06T16:23:16.567+0100] {logging_mixin.py:190} INFO - [2025-01-06T16:23:16.567+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-06T16:23:16.652+0100] {logging_mixin.py:190} INFO - [2025-01-06T16:23:16.641+0100] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-06T16:23:16.653+0100] {logging_mixin.py:190} INFO - [2025-01-06T16:23:16.653+0100] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-06T16:23:16.653+0100] {processor.py:927} WARNING - No viable dags retrieved from /Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-06T16:23:16.837+0100] {processor.py:208} INFO - Processing /Users/azeez.ganiyu/airflow_venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.289 seconds
